web-client/src/presenter/sequences/gotoDashboardSequence.ts

```diff
@@ -115,7 +115,7 @@ export const gotoDashboardSequence = [
           setDefaultCaseTypeToDisplayAction,
           getOpenAndClosedCasesForUserAction,
           setCasesAction,
-          setupCurrentPageAction('DashboardPractitioner'),
+          setupCurrentPageAction('DashboardPetitioner'),
```

Comment:
Out of curiosity, why did this change? The tests that reference it are still called "practitioner"

Reply:
After reading further, I see that we now only have one dashboard for petitioners and practitioners, nice! How do you feel about giving the page a more generic name?

Reply:
That is a good idea. I'll update. Thanks!